### DISCUSSION TITLE
将界面翻译成了中文，替换文件就能用啦，希望能降低一些使用门槛：）

### DISCUSSION THREAD

**Discussion by zhuhehuai** [oobabooga/text-generation-webui#1181] [Show and tell]
下载替换就能用了
[text-generation-webui.zip](https://github.com/oobabooga/text-generation-webui/files/11233677/text-generation-webui.zip)

**Comment by alex13by** [comment]
![Uploading image.png…]()

**Comment by jinwukong** [comment]
大佬还在更新吗，替换重复文件后会提示报错

**Comment by kingck9457** [comment]
大老能更新一下嗎!@!

**Comment by Yang-Yekun** [comment]
> 下载替换就能用了 [text-generation-webui.zip](https://github.com/oobabooga/text-generation-webui/files/11233677/text-generation-webui.zip)

大佬这怎么替换
